test/integration/apiserver/admissionwebhook/admission_test.go

```diff
@@ -127,9 +133,16 @@ var (
 		gvr("authorization.k8s.io", "v1beta1", "selfsubjectaccessreviews"):  sets.NewString("*"),
 		gvr("authorization.k8s.io", "v1beta1", "selfsubjectrulesreviews"):   sets.NewString("*"),
 
-		// TODO: webhook config objects are not subject to admission, verify CRUD works and webhooks do not observe them
-		gvr("admissionregistration.k8s.io", "v1beta1", "mutatingwebhookconfigurations"):   sets.NewString("*"),
-		gvr("admissionregistration.k8s.io", "v1beta1", "validatingwebhookconfigurations"): sets.NewString("*"),
+		// TODO: implement custom subresource tests (requires special states or requests)
```

Comment:
drop these lines (they're in customTestFuncs now)

Reply:
I do not see these lines in my code by some reason..

Reply:
nvm I see what you meant, done..